.github/workflows/AFTER_MERGE.yml

```diff
@@ -0,0 +1,24 @@
+name: AFTER_MERGE
+on:
+  push:
+    branches:
+    - "master"
+
+jobs:
+  Create_pull_request:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Create pull request to develop
+      uses: repo-sync/pull-request@65194d8015be7624d231796ddee1cd52a5023cb3
+      with:
+        source_branch: ""                                   # If blank, default: triggered branch
+        destination_branch: "develop"                       # If blank, default: master
+        pr_title: "Merge master to develop"                 # Title of pull request
+        pr_body: "This pulls ${{ github.ref }} to develop." # Full markdown support, requires pr_title to be set
```

Comment:
Just because we're on the web we could actually link here :see_no_evil:.

Reply:
What would you expect?

Reply:
I'd love to add a pull request here. However, we react to commit being added. As a result, I cannot give a link to a PR. Also note that not all commits on main/master come from PRs.

Reply:
I'd expect simple linking to happen

```
This pulls [`${{ github.ref }}`](https://github.com/barmac/test/tree/${{ github.ref }}`) into [`develop`](https://github.com/barmac/test/develop).
```

Reply:
Alternatively we could simplify the message:

* "This integrates latest changes from master."

Reply:
Let's use the simplified message. Thanks for the suggestion.

Reply:
One issue is that we do not currently allow merge as a strategy in our PRs. We'd need to adjust that accordingly.

Also we need to make sure to _only allow merge_ as everything else will screw up our history (i.e. prevent us from merging `develop` into `master` without actual commit duplication).

Reply:
As I think about it I'd rather love our [https://github.com/bpmn-io-bot](https://github.com/bpmn-io-bot) to automatically merge and if that fails shoot us a message via slack.

Reply:
Cool idea. I'll try that out.